vulnerabilities/pipes/advisory.py

```diff
@@ -192,6 +188,29 @@ def insert_advisory_v2(
                 level=logging.ERROR,
             )
 
+    if created:
```

Comment:
How do we ensure that if the loop breaks before creating all ImpactedPackages in the list, then also we are able to create all ImpactedPackages when the importer restarts.

Reply:
We now use atomic transactions to ensure that partial advisory is never imported.